src/app/components/add-content/add-content.component.ts

```diff
@@ -64,9 +71,11 @@ export class AddContentComponent implements OnInit {
   }
 
   uploadContent() {
+    this.content.organization = 'ברירת מחדל';

```

Comment:
Why 'ברירת מחדל'? Should it come from the user's organization?

Reply:
You said yes.

Reply:
It will be fixed after merge with another branch that deals with user registration.